homeassistant/components/samsungtv/bridge.py

```diff
@@ -34,13 +37,45 @@
 )
 
 
+def mac_from_device_info(info):
+    """Extract the mac address from the device info."""
+    if info is not None:
```

Comment:
Consider moving this check to the caller. We already check it in two out of three call places.

Reply:
Will do when I break this up now that I've finished the functional testing